### ISSUE TITLE
Some notes from build experiences

### ISSUE THREAD

**Issue by ghost** [TomWhitwell/Magnetophon#10]
Hi not actually an issue but it does address several existing issues (open and closed) and therefore might be of use to some.

I've successfully built this module using these tape-heads - https://www.aliexpress.com/item/33005387634.html?spm=a2g0s.12269583.0.0.513062e8RFe37r

To fit in the panel, I removed the metal guide/fork and then bent the rest of the metal around the tape head body. I soldered trimmed off component legs to the tape head legs, and then mounted in the PCB/panel before soldering. The bent around metal frame helps it sit in place but I'm going to use a dab of epoxy to make it a bit more solid.

I did have no end of trouble at first. It didn't *sound* like oscillation but the extra capacitor definitely helped.

There is a little interference/hum when holding tape near the head but it's acceptable.

For the tape wand, I again removed the guide forks and bent the rest of the metal around the tape head. I used stereo/2-core shielded cable, as recommended in an issue above. Each core was soldered to a tape head leg with the shield trapped and soldered between the extra-bent-around metal and the tape head. I then used excessive amounts of heat-shrink (descending in size to fit both the head and cable) to keep it all secure, neat, insulated, etc. On the other end, a 3.5mm mono jack plug with one of the cores wired to the tip and both the other core and shield wired to the sleeve connection.

Basically, the wand was unusable when I first tried with mono cable but works fine with a stereo cable. Again, a bit of hum when actually holding the tape head but it's all workable.

Anyway, hope this helps others and thanks for making the module open source etc.
